src/WhatToCook.Application/Domain/Recipe.cs

```diff
@@ -1,20 +1,21 @@
-﻿
-namespace WhatToCook.Application.Domain;
+﻿namespace WhatToCook.Application.Domain;
 
 public class Recipe
```

Comment:
Missing constructor and encapsulation (important setters are public)

Reply:
done, will be in the next commit